flycheck-drstring.el

```diff
@@ -41,10 +29,25 @@
 ;; Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
 ;; Boston, MA 02110-1301, USA.
 
+;;; Commentary:
+
+;; This package adds support for linting documentation in Swift files
+;; using DrString (https://github.com/dduan/DrString/).  To configure
+;; this package, add the following line to your initialization file:
+
+;; (flycheck-drstring-setup)
+
+;; DrString is a CLI tool that can be installed using your favorite
+;; package manager.  For example, if you use Homebrew:
+
+;; $ brew install dduan/formulae/drstring
+
 ;;; Code:
 
 (require 'flycheck)
 
+(declare-function swift-mode "swift-mode")
```

Comment:
I think we can simply `(require 'swift-mode)` and add it as a package dependency. It's the easiest approach.

Reply:
Yes, I agree.
Fix branch.